src/Ng/Jobs/MonitoringProcessorJob.cs

```diff
@@ -29,6 +29,7 @@ public class MonitoringProcessorJob : LoopingNgJob
     {
         private const int DefaultQueueLoopDurationHours = 24;
         private const int DefaultQueueDelaySeconds = 30;
+        private static readonly TimeSpan MaxShutdownTime = TimeSpan.FromMinutes(1);
```

Comment:
Why do we pick up 1 minute here?

Reply:
I mimicked the time we picked for our subscription processor job. This was pretty arbitrary. I wouldn't want to wait too much longer than this because this window of time is essentially downtime in the "hung" state that we've seen.